__tests__/TimerMixin-test.js

```diff
@@ -46,8 +47,12 @@ describe('TimerMixin', function() {
       global[type.clearer].mockClear();
       var cb = jest.genMockFunction();
       var id = component[type.setter](cb, 10);
-
-      expect(global[type.setter]).toBeCalledWith(cb, 10);
```

Comment:
This was failing because the global setter is never actually called with the callback function. It's called with a wrapper function that calls `callback.apply(this, arguments)`